src/client/utils/dom.js

```diff
@@ -647,15 +655,15 @@ export function isDocumentFragmentNode (el) {
 }
 
 export function isAnchorElement (el) {
-    return getTagName(el) === 'a' && isDomElement(el);
+    return instanceToString(el) === '[object HTMLAnchorElement]';
 }
 
 export function isTableElement (el) {
-    return getTagName(el) === 'table' && isDomElement(el);
+    return instanceToString(el) === '[object HTMLTableElement]';
 }
 
-export function isTableDataElement (el) {
-    return getTagName(el) === 'td' && isDomElement(el);
+export function isTableCellElement (el) {
+    return instanceToString(el) === NATIVE_TABLE_CELL_STR;
```

Comment:
In standart it named as `TableDataCell`.
https://www.w3.org/TR/html401/struct/tables.html#h-11.2.6

`TableCell` is common name.
There are th - table header cell, td - table data cell.
So we should rename it to the `isTableDataCellElement`.